simapp/app.go

```diff
@@ -289,7 +289,7 @@ func NewSimApp(
 	bApp.SetParamStore(&app.ConsensusParamsKeeper)
 
 	// add keepers
-	app.AccountKeeper = authkeeper.NewAccountKeeper(appCodec, keys[authtypes.StoreKey], authtypes.ProtoBaseAccount, maccPerms, sdk.Bech32MainPrefix, authtypes.NewModuleAddress(govtypes.ModuleName).String())
+	app.AccountKeeper = authkeeper.NewAccountKeeper(appCodec, runtime.NewKVStoreService(keys[authtypes.StoreKey]), authtypes.ProtoBaseAccount, maccPerms, sdk.Bech32MainPrefix, authtypes.NewModuleAddress(govtypes.ModuleName).String())
```

Comment:
Same comment as before about UPGRADING.md + changelog.
I think now the category can be generalized under simapp. Moreover, we should add that if people are generating mocks, they will need to re-generate them.

Reply:
Github is taking a while to update stuff here, but I've added a changelog entry and modified the upgrading doc.